services/api-server/src/simcore_service_api_server/api/routes/_constants.py

```diff
@@ -0,0 +1,24 @@
+from typing import Final
+
+#
+# CHANGELOG formatted-messages for API routes
+#
+# - Append at the bottom of the route's description
+# - These are displayed in the swagger doc
+# - These are displayed in client's doc as well (auto-generator)
+# - Inspired on this idea https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#describing-changes-between-versions
+#
+
+# new routes
+FMSG_CHANGELOG_NEW_IN_VERSION: Final[str] = "New in *version {}*\n"
```

Comment:
what means FMSG?

Reply:
the concept of `f-string`  we know, right?
These as messages i.e. MSG and `FMSG` suggest `f-messages` or formatted messages